formbody.d.ts

```diff
@@ -2,6 +2,7 @@ import { FastifyPlugin } from 'fastify'
 
 export interface FormBodyPluginOptions {
   bodyLimit?: number
+  parser?: (str: string) => string
```

Comment:
I think return type is incorrect, for example default parser returns dictionary.
I would consider to change it to unknown

Reply:
How about returning `Record<string, unknown>`?

Reply:
Both looks applicable for me, not sure what is the best approach